README.md

```diff
@@ -76,7 +76,7 @@ rustup target add wasm32-wasip2
 # Install required cargo components
 # https://github.com/bytecodealliance/cargo-component#installation
 cargo install cargo-binstall
-cargo binstall cargo-component warg-cli wkg --locked --no-confirm --force
+cargo binstall cargo-component wasm-tools warg-cli wkg --locked --no-confirm --force
```

Comment:
What is `wasm-tools` being used for here?

Reply:
I can remove it from this PR, but I was following the instruction on a fresh machine and this one was missing, blocking the rest of the instructions.